openupgrade_scripts/scripts/mail/17.0.1.15/post-migration.py

```diff
@@ -10,6 +10,8 @@
     "mail.mail_channel_rule",
     "mail.channel_all_employees",
     "mail.channel_member_general_channel_for_admin",
+    "mail_activity_unlink_log.mt_activities_unlink",
+    "mail_activity_unlink_log.message_activity_unlink",
```

Comment:
```suggestion
    "mail_activity_cancel_tracking.mt_activities_unlink",
    "mail_activity_cancel_tracking.message_activity_unlink",
```

Reply:
Oh, then this should be added in mail_activity_cancel_tracking module instead, right?

Reply:
Yes